src/components/navbar/Navbar.tsx

```diff
@@ -113,19 +115,23 @@ function Navbar() {
             <Link to={CLIENT_PROFILE}>{t("Profile.title")}</Link>
           </Menu.Item>
         ) : null}
-        {navItems?.options.map(({ links, title }) => (
-          <Menu.SubMenu
-            icon={verifyMenuItemIcon(title)}
-            key={title}
-            title={title === t("Chat.title") ? "" : title}
-          >
-            {links.map((link) => (
-              <Menu.Item key={link.to} icon={badgeForChatSubMenu(link.title)}>
-                <Link to={link.to}>{link.title}</Link>
-              </Menu.Item>
-            ))}
-          </Menu.SubMenu>
-        ))}
+        {navItems?.options.map(({ links, title }) =>
+          title ? (
+            <Menu.SubMenu
+              icon={verifyMenuItemIcon(title)}
+              key={title}
+              title={title === t("Chat.title") ? "" : title}
+            >
+              {links.map((link) => (
+                <Menu.Item key={link.to} icon={badgeForChatSubMenu(link.title)}>
+                  <Link to={link.to}>{link.title}</Link>
+                </Menu.Item>
+              ))}
+            </Menu.SubMenu>
+          ) : (
+            <li style={{ flexGrow: 1, order: 2 }} />
```

Comment:
Is this so you can position the chat icon? Can you try moving the chat icon to the NavBarButtons section to avoid empty objects and invisible conditional render?

Reply:
fixed